File: Assignments/A14/12577/12577.cpp
#include <iostream>
using namespace std;
int main(){
    string str;
    int caseNumber=1;
    while(cin >>str){
        if(str =="Hajj")
            cout << "Case " << caseNumber << ": Hajj-e-Akbar" << endl;
        else if(str == "Umrah")
            cout << "Case " << caseNumber << ": Hajj-e-Asghar" << endl;
        else if(str =="*")
            break;
        caseNumber++;
    }
    return 0;
}

File: Assignments/A14/12577/12577.py
import sys
x=1
for line in sys.stdin:
    if line == 'Hajj':
        print(f'Case {x}: Hajj-e-Akbar')
    elif line == 'Umrah':
        print(f'Case {x}: Hajj-e-Asghar')
    elif line == '*':
        breakpoint
    x+=1